test/recurly/BaseClient.test.js

```diff
@@ -228,7 +228,7 @@ describe('BaseClient', () => {
         .listResources()
         .count()
         .catch(err => {
-          assert(err instanceof recurly.errors.ValidationError)
+          assert(err instanceof recurly.errors.UnprocessableEntityError)
```

Comment:
If it is no longer throwing a validationerror, we should update the description of this test